OData/test/UnitTest/System.Web.OData.Test/OData/ResourceContextTest.cs

```diff
@@ -224,6 +224,34 @@ public void Property_ResourceInstance_EdmObjectHasCollectionProperty_ForComplexC
         }
 
         [Fact]
+        public void Property_ResourceInstance_HandlesModelClrNameDifferences()
+        {
+            const string ClrPropertyName = "Property";
+            const string ModelPropertyName = "DifferentProperty";
+
+            // Arrange
```

Comment:
Does it make sense to have test specific constants defined for the entire class? Maybe for the "ClrPropertyName" but probably not for the "ModelPropertyName". I could always just leave them inline as well :)